specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2022-11-30-preview/definitions/MigrateMySqlAzureDbForMySqlOfflineTask.json

```diff
@@ -46,13 +46,58 @@
           "format": "date-time",
           "description": "Parameter to specify when the migration started"
         },
+        "sourceServerResourceId": {
+          "type": "string",
+          "description": "Optional resource Id of the source server if it is an azure instance"
+        },
+        "targetServerResourceId": {
+          "type": "string",
+          "description": "Optional resource Id of the target server"
+        },
         "optionalAgentSettings": {
           "type": "object",
-          "description": "Optional parameters for fine tuning the data transfer rate during migration",
+          "description": "Optional parameters for fine tuning the data migration",
+          "properties": {
+            "enableConsistentBackup": {
+              "type": "boolean",
+              "description": "Setting to enable transactional consistency",
+              "default": false
```

Comment:
Does this need to be true since it's true by default in the portal?

Reply:
The default value of the EnableConsistentBackupSetting in the backend code is false, I've just checked. It is true in the portal because it assigns it explicitly. But, if we do not specify it in the request, it will be false.